challenge.md

```diff
@@ -32,19 +32,20 @@ Open up this file in your text editor, and write your answers below (where indic
 1. Is `/Users/blah_blah/Desktop` a relative path or an absolute path? How do you know?
 
  <!-- Answer Starts Here -->
- 
+ It is an absolute path because it is relative to the root directory and it starts with a forward slash
+
  <!-- Answer Ends Here -->
 
 2. Suppose that we're working on a project, and we want to use a font that's being hosted somewhere on the internet. Would we use an absolute or relative path to refer to it? Why?
 
  <!-- Answer Starts Here -->
- 
+ You would use an absolute path because it shows the location of the file within the system
  <!-- Answer Ends Here -->
 
 3. Now suppose that we have an image file living inside our project. Would we refer to it with an absolute or relative path? Why?
 
  <!-- Answer Starts Here -->
- 
+ Relative path because it is nested within the project folder.
```

Comment:
OK, thanks for the feedback!
On Sep 28, 2015 5:35 PM, "Matt Brendzel" <notifications@github.com> wrote:

> In challenge.md
> <https://github.com/ga-wdi-boston/unix-cli-intro-assessment/pull/20#discussion_r40608410>
> :
>
> >   <!-- Answer Ends Here -->
> >
> >  3. Now suppose that we have an image file living inside our project. Would we refer to it with an absolute or relative path? Why?
> >
> >   <!-- Answer Starts Here -->
> > -
> > + Relative path because it is nested within the project folder.
>
> Yes, this is true. But the real reason is that we want to prevent links
> from breaking when we move our project folder.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ga-wdi-boston/unix-cli-intro-assessment/pull/20/files#r40608410>
> .
>